test/test_nn.py

```diff
@@ -10116,6 +10115,145 @@ def test_grid_sample_3d_alert_nondeterministic(self, device):
         output = F.grid_sample(input, grid, align_corners=False)
         output.sum().backward()
 
+    @dtypes(torch.float, torch.double)
+    @largeTensorTest(lambda self, device, dtype:
+                     32769 * (65536 + 3 * 65536 / 128) *
```

Comment:
Where does this number come from? Looking at the test, I would have expected:

- for the im tensor: 32769 * 65536 * element_size
- for the `small_image` tensor: 32769 * element_size

Everything else looks pretty small

Reply:
A comment for how we got `32769 * (65536 + 3 * 65536 / 128) * torch.tensor([], dtype=dtype).element_size()` would be nice for future devs looking to modify the test